crates/ffi/YttriumCore/Sources/YttriumCore/Config.swift

```diff
@@ -78,35 +78,13 @@ extension Endpoints {
     }
 }
 
-import SwiftDotenv
 
 extension Config {
     
     public static func local() -> Self {
         Config(endpoints: .local())
     }
     
-    static func pimlico() throws -> Self {
```

Comment:
Are this config logic getting moved elsewhere?

Reply:
it wasn't used at all, we are injecting bundlerUrl from swift on init